components/uuf-core/src/main/java/org/wso2/carbon/uuf/internal/RequestDispatcher.java

```diff
@@ -59,6 +66,9 @@ public RequestDispatcher() {
     public RequestDispatcher(StaticResolver staticResolver, Debugger debugger) {
         this.staticResolver = staticResolver;
         this.debugger = debugger;
+
+        // Add filters
+        filters = new ArrayList<>(Arrays.asList(new CsrfFilter()));
```

Comment:
No need to convert the List again to an ArrayList, `Arrays.asList` will be enough here.